package.json

```diff
@@ -82,8 +84,9 @@
     "@types/redis": "^2.8.14",
     "@types/sinon": "^7.5.0",
     "bull": "*",
-    "dd-trace": "*",
+    "dd-trace": "^0.30.5",
```

Comment:
did you intend to change this with this pr?

Reply:
No, nice catch!
I am reverting it even though it is devDependencies (It won't affect apps using orka)